money-core/src/main/resources/reference.conf

```diff
@@ -39,6 +39,8 @@ money {
     jvm-exit-on-fatal-error = off
   }
 
+  metricRegistryFactory = "com.comcast.money.metrics.DefaultMetricRegistryFactory"
```

Comment:
maybe put this in its own subheading..

metrics-registry {
     class-name = "..."
}